go.mod

```diff
@@ -1,30 +1,35 @@
 module github.com/Decentr-net/vulcan
 
-go 1.15
-
-replace github.com/docker/docker => github.com/docker/engine v0.0.0-20190717161051-705d9623b7c1 // fix logrus for testcontainers
+go 1.16
```

Comment:
Have you updated Dockerfile?

Reply:
there is no necessity. It was a fix for `testcontainers` package